projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -1490,6 +1495,9 @@
 }
 
 ONNX_XFAIL_SET = {
+    # Failure - inconsistent initialization
+    "LSTMModule_basic",
```

Comment:
If the test is just failing on everything its not worth including yet. We can include a comment that we tested this via a downstream test suite.